library/src/main/java/com/google/android/exoplayer2/extractor/mkv/MatroskaExtractor.java

```diff
@@ -1608,4 +1628,118 @@ private static boolean parseMsAcmCodecPrivate(ParsableByteArray buffer) throws P
 
   }
 
+  private class TrueHDSampleHolder {
```

Comment:
I'll look into seeing if this can be simplified more.  Previous attempts at doing batching in AudioTrack didn't work.